Index: packages/sqlite/src/sqlite3db.pas
===================================================================
--- packages/sqlite/src/sqlite3db.pas	(revision 22896)
+++ packages/sqlite/src/sqlite3db.pas	(working copy)
@@ -314,6 +314,7 @@
    end;
    if length(InterS) > 0 then Field.add(InterS);
    List_Field.add(Field);
+   Field.Free;
 end;
 {*************************************************************}
 constructor TSQLite.Create(DBFileName: String);
@@ -348,6 +349,7 @@
       fError := SQLITE_OK;
    end;
    fMsg := sqlite3_errmsg(fSQLite);
+   strdispose(name);
 end;
 {*************************************************************}
 destructor TSQLite.Destroy;
